eip-0031.md

```diff
@@ -0,0 +1,340 @@
+Babel Fees
+============================================
+
+* Author: nitram147
+* Status: Proposed
+* Created: 25-January-2022
+* Last edited: 25-January-2022
+* License: CC0
+* Forking: not needed
+
+Introduction
+--------------------------------------------
+
+The term “babel fees“ refers to the concept of paying transaction fees in tokens (fe. stablecoins) instead of platform’s primary token (ERG). For more information about the origin of the term “babel fees“, please see the following IOHK article:
+[https://iohk.io/en/blog/posts/2021/02/25/babel-fees/](https://iohk.io/en/blog/posts/2021/02/25/babel-fees/)
```

Comment:
The purpose of this part of text (to which you left your comment), is to give credit to the place from which I've got inspiration for the name of this purpose. And since this EIP concept has the same name as the similar concept on Cardano, I thought it'd be appropriate to explain the difference to the EIP reader. Should I also mention somewhere in the EIP that the similar idea was implemented in ErgoMixer for mixing transactions fee payments?

Reply:
I guess that it would be reasonable to mention ErgoForum ideas and ErgoMixer, to give credits to their authors, and also there are babel fees variants there which can be useful in some use-cases.